### PR TITLE
LPD-15335 Implement Functional Test to Validate Deletion of Object Relationships From Different Folders

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPD-15368)

### CODE REVIEW

**Comment by igor-franca** [comment]
I believe that the CI bot added the labels incorrectly. So I'm removing some of them:

File: modules/test/playwright/tests/object-web/modelBuilder.spec.ts [hunk 1]
```diff
@@ -86,3 +86,85 @@ test('can create relationship by dragging node handles', async ({
 
 	await apiHelpers.objectAdmin.deleteObjectFolder(objectFolder.id);
 });
+
+test('can delete object relationship from different folders', async ({
+	apiHelpers,
+	modelBuilderPage,
+	objectDefinitionsPage,
+}) => {
+	await apiHelpers.featureFlag.updateFeatureFlag('LPS-148856', true);
```
Comment (rebsilva) [inline modules/test/playwright/tests/object-web/modelBuilder.spec.ts:95]: i think the FF for the model builder was already removed right? can we delete this line?

**Review by rebsilva — COMMENTED** [review]
@gabrielwas can you also take a look on this PR please? i can make the changes necessary to forward this :cherry_blossom:

**Comment by rebsilva** [comment]
follow-up: https://github.com/liferay-objects/liferay-portal/pull/3094

**Pull request closed without merging**